TimberAPI/SpecificationSystem/Buildings/RecipeDeserializer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Timberborn.Persistence;
+
+namespace TimberbornAPI.SpecificationSystem.Buildings
+{
+    public class RecipeDeserializer : IObjectSerializer<Recipe>
+    {
+        private readonly PropertyKey<string> _recipeId = new PropertyKey<string>("RecipeId");
+
+        /// <summary>
+        /// This class only deserializes specification jsons, so this is not used
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="objectSaver"></param>
+        public void Serialize(Recipe value, IObjectSaver objectSaver)
+        {
+            throw new NotSupportedException();
+        }
+
+        public Obsoletable<Recipe> Deserialize(IObjectLoader objectLoader)
+        {
+            var recipe = objectLoader.Has(_recipeId)
```

Comment:
Fallback :D

Reply:
Same as on others, was done because it prevented crashes